compiler/rustc_error_codes/src/error_codes/E0665.md

```diff
@@ -16,18 +16,30 @@ The `Default` cannot be derived on an enum for the simple reason that the
 compiler doesn't know which value to pick by default whereas it can for a
 struct as long as all its fields implement the `Default` trait as well.
 
-If you still want to implement `Default` on your enum, you'll have to do it "by
-hand":
+For the case where the desired default variant has no data, you can annotate
```

Comment:
```suggestion
For the case where the desired default variant has no payload, you can annotate
```

Minor, semantics: "Data" is pretty vague and ambiguous. For example, you are allowed to place `#[default]` on a (unit) variant with an explicit discriminant like `Var = 16`. Arguably, `Var` does have *data*. I'd prefer we use the term *payload* here or rephrase it slightly to mention *unit variant* instead.